pkg/e2e/e2e.go

```diff
@@ -401,6 +403,28 @@ func runTestsInPhase(phase string, description string) bool {
 		return false
 	}
 
+	h := helper.NewOutsideGinkgo()
+	dependencies, err := debug.GenerateDependencies(h.Kube())
+	if err != nil {
+		log.Printf("Error generating dependencies: %s", err.Error())
+	} else {
+		if len(dependencies) > 0 {
+			err = ioutil.WriteFile(filepath.Join(phaseDirectory, "dependencies.txt"), []byte(dependencies), 0644)
+		}
+
+		if cfg.JobName != "" && cfg.JobID != "" {
+			diff, err := debug.GenerateDiff(cfg.JobName, cfg.JobID, dependencies, phase)
```

Comment:
This should probably be JobID - 1, shouldn't it?

Reply:
Yeah that was a derp on my part.